js/.storybook/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useFonts } from '../packages/messenger-app/fonts-loader'
+import { mockServices } from '../packages/utils/testing/mockServices.test'
+
+import StorybookUIRoot from './Storybook'
+
+mockServices()
+
+const App: React.FC = () => {
+	const { isFontLoaded } = useFonts()
```

Comment:
Is it possible to have a wrapper, that add all common providers between the app and storybook ?
And then two differents wrapper, one for the app and one for the storybook, that add specific providers ?

Reply:
Its not really out of context because if we dont implement storybook there is no reason to do this refactor 👍 
But if you want to do this in the next PR no problem 👍